modules/restheart/scripts/src/db.converters.ts

```diff
@@ -1,28 +1,20 @@
-/**
- * Created by nitzan on 02/01/2017.
- */
-
-/// <reference path="../../../../scripts/bin/app/application.d.ts" />
-/// <reference path="../../../../scripts/bin/components/components.d.ts" />
-/// <reference path="../../../../scripts/bin/components/converters.d.ts" />
-
-(function(): void {
-	fugazi.components.modules.descriptor.loaded({
+(function (): void {
+	fugazi.loaded({
 		name: "restheart.databases",
 		converters: {
 			RawDatabases2Databases: {
 				title: "converts RawDatabases to Databases",
 				input: "RawDatabases",
 				output: "Databases",
-				converter: function(response: { _embedded: { _id: string }[] }): string[] {
+				converter: function (response: { _embedded: { _id: string }[] }): string[] {
```

Comment:
can this space addition be removed?
it also creates unimportant changes in a big PR